### PR TITLE
Update FrontDoor WAF API to 2019-03-01-preview

### PR DESCRIPTION
**Pull request by eschwabe**
* Generated new SDK from swagger
* Updated unit tests to use new swagger definition
* Incremented Microsoft.Azure.Management.FrontDoor package version

This PR supersedes the automated PR [5527](https://github.com/Azure/azure-sdk-for-net/pull/5527).

### CODE REVIEW

File: src/SDKs/FrontDoor/Management.FrontDoor/Microsoft.Azure.Management.FrontDoor.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.FrontDoor</PackageId>
 		<Description>Provides management capabilities for Front Door services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.FrontDoor</AssemblyName>
-		<Version>0.11.0-preview</Version>
+		<Version>0.12.0-preview</Version>
 		<PackageTags>Microsoft Azure Front Door management;FrontDoor;Front Door management;</PackageTags>    
 		<PackageReleaseNotes>
 			Fixed a regression in enable/disable https for custom domain.
```
Comment (dsgouda) [inline src/SDKs/FrontDoor/Management.FrontDoor/Microsoft.Azure.Management.FrontDoor.csproj]: Please update packagereleasenotes

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please use [generate.ps1](https://github.com/Azure/azure-sdk-for-net/blob/f716788ea577ccb37c3ef58ba1f756b4197bbc20/src/SDKs/FrontDoor/Management.FrontDoor/generate.ps1) to generate the code and commit all changes

File: src/SDKs/FrontDoor/Management.FrontDoor/Microsoft.Azure.Management.FrontDoor.csproj (hunk 1 above)
Comment (eschwabe) [reply src/SDKs/FrontDoor/Management.FrontDoor/Microsoft.Azure.Management.FrontDoor.csproj]: Updated release notes.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**